config/sync/views.view.content_paragraphs.yml

```diff
@@ -0,0 +1,168 @@
+uuid: 66163bca-af6b-4db4-b820-3579418763ea
+langcode: en
+status: true
+dependencies:
+  config:
+    - core.entity_view_mode.node.card
+  module:
+    - node
+    - user
+id: content_paragraphs
+label: 'Content paragraphs'
+module: views
+description: ''
+tag: ''
```

Comment:
Consider adding more information here.

To me this title and description is fairly non-descript but it is heavily tied to the `dpl_filter_paragraphs` module. Can we update it to make this relationship more clear?